python/OpenVINO_EP/yolov4_object_detection/README.md

```diff
@@ -16,47 +16,63 @@ The source code for this sample is available [here](https://github.com/microsoft
 # How to build
 
 ## Prerequisites
-1. [The Intel<sup>®</sup> Distribution of OpenVINO toolkit](https://docs.openvino.ai/latest/openvino_docs_install_guides_install_runtime.html)
-   Please select Install OpenVINO Runtime using an installer 
-2. Please check also the documentation link for the [installer](https://docs.openvino.ai/latest/openvino_docs_install_guides_installing_openvino_linux.html#doxid-openvino-docs-install-guides-installing-openvino-linux)
-3. Download the latest version of the [YOLOv4](https://github.com/onnx/models/tree/master/vision/object_detection_segmentation/yolov4) model from here.
-
-## Install ONNX Runtime for OpenVINO Execution Provider
-Please install the onnxruntime-openvino python package from [here](https://github.com/intel/onnxruntime/releases/)
+1. For Windows, [The Intel<sup>®</sup> Distribution of OpenVINO™ toolkit](https://docs.openvino.ai/latest/openvino_docs_install_guides_installing_openvino_windows_header.html#doxid-openvino-docs-install-guides-installing-openvino-windows-header).
+   Please select Install OpenVINO™ from PyPI.
+   ```
+   pip3 install openvino
+   ```
+2. Download the latest version of the [YOLOv4](https://github.com/onnx/models/tree/master/vision/object_detection_segmentation/yolov4) model from here.
+
+## Install ONNX Runtime for OpenVINO™ Execution Provider
```

Comment:
Please add documentation that it contains prebuilt openvino libs with ABI Flag 0